posts/2023-08-30_pharmaverse_story/pharmaverse_story.qmd

```diff
@@ -0,0 +1,94 @@
+
+So, again, companies re-started to working in silos to find those answers.
+But now, in collaborative silos and with common goals: create extremely useful packages to solve pharmaceutical-specific gaps once and solve them well!
+
+From Stackhouse's report, Michael Rimler and he talked and ultimately formed a partnership between GSK and Atorus to develop a few more packages, including [*metacore*](https://github.com/atorus-research/metacore), [*xportr*](https://github.com/atorus-research/xportr), and *timber* (which became [*logrx*](https://github.com/pharmaverse/logrx).
+
+Around the same time, Thomas Neitmann (actual Denali Therapeutics, ex-Roche) and Rimler discovered that both were working with ADaM in R, so Ross and Rimler saw an opportunity there and GSK started their partnership with Roche to build and release [*admiral*](https://github.com/pharmaverse/admiral) package.
+
+The idea of working together, the sense of community, and the appetite from organizations built more and more, with incentive and priority established up into the programming heads council.
+
+Janssen had a huge effort in building R capabilities going on as well, by releasing [*tidytlg*](https://github.com/pharmaverse/tidytlg) and [*envsetup*](https://github.com/pharmaverse/envsetup)), so eventually Rimler, Stackhouse and Ross formalized [pharmaverse](https://pharmaverse.org/) and formed the council, adding in Sumesh (Janssen).
+
+At the end of their presentation ["Closing the Gap: Creating an End to End R Package Toolkit for the Clinical Reporting Pipeline."](https://www.youtube.com/watch?v=NW9qp5QOlIU), in [R/Pharma 2021](https://rinpharma.com/event/rinpharma2021/), Ben Straub's (GSK) and Eli Miller's (Atorus) welcomed the community to the [pharmaverse](https://pharmaverse.org/), a new group of packages developed by various pharmaceutical companies to support open-source clinical workflows.
+
+From the outset, the name [pharmaverse](https://pharmaverse.org/) was chosen so that it could be a neutral home, unrelated to any company.
+Also, it was established as not being a consortium, which means that founders don't own, fund, or maintain the packages.
+Some individuals and companies maintain them but don't take ownership of them.
+The focus of [pharmaverse](https://pharmaverse.org/) early on, and today, is on inter organization cooperation, to build an environment where, if organizations identify that they have a joint problem that they want to solve, this is the right space to work on and release it.
+
+[Pharmaverse](https://pharmaverse.org/) has grown a lot, at the time of writing this post we have 75 packages hosted in [pharmaverse](https://pharmaverse.org/) [CRAN-like instance](https://pharmaverse.r-universe.dev/builds), and this has led to a partnership with [PHUSE](https://phuse.global/About_PHUSE) to get support from their organization and platform, and because they are eager to advance and support [pharmaverse](https://pharmaverse.org/) mission.
```

Comment:
```suggestion
[Pharmaverse](https://pharmaverse.org/) has grown a lot, at the time of writing this post we have >25 packages recommended in [pharmaverse](https://pharmaverse.org/e2eclinical/), and this has led to a partnership with [PHUSE](https://phuse.global/About_PHUSE) to get support from their organization and platform, and because they are eager to advance and support [pharmaverse mission](https://pharmaverse.org/charter/).
```
See https://github.com/pharmaverse/pharmaverse/issues/252 as FYI - i kept the exact number a bit looser as it changes all the time.